.github/workflows/pr-mirror-repo-sync.yml

```
@@ -0,0 +1,35 @@
+name: PR Mirror and Repo Sync
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)